project-management/engagement-guides/meet-up-sessions/invite-template.md

```diff
@@ -0,0 +1,132 @@
+* You can leave at any point if you want without giving a reason. 
+* You can rejoin again whenever you like.
+* You can take part by writing (Zoom chat or google doc), speaking, or both. If you prefer to just type in the shared document, that is okay and there is no pressure to speak. If you prefer to speak and not write, that is okay too and we will provide a scribe to make sure your input is captured and included. 
+* If you are not talking, please keep your microphone muted, and if you would to talk, please raise your hand.
+* If you prefer to keep your camera and microphone switched off and just listen, that is completely fine.  
+* We will keep notes in a shared [Google Doc] - link to week's doc - ()
+
+## Agenda
+
+| Time (GMT) | Activity |
+| ---- | -------- |
+| 12:30pm | 👋 Welcome, [code of conduct review](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/.github/CODE_OF_CONDUCT.md) |
+| 12:35pm | Introductions and goal-setting |
+| 12:40pm | Breakout room discussion 1|
+| 1pm | ☕️ Break |
+| 1:05pm | Breakout room discussion 2 |
+| 1:35pm | ☕️ Break  |
+| 2pm | All return to main room for group sharing/discussions |
+| 2:25pm | Open discussion: celebrations, reflections and future directions |
+| 2:30pm | 👋 Close |
+
+### Shared timer for this call
+
+[https://cuckoo.team/autspaces-meetup](https://cuckoo.team/autspaces-meetup)
+*This link will allow you to keep track of different session.*
+
+## Breakout Room Groups and Activities: Topic proposals
+
+*If you have an idea for a topic you'd like to discuss or work on during the discussion session (either in the main room or in a breakout room), please add it below and put your name next to it. If you like one of the topics that are already suggested, please add your name next to that one. Teamwork makes the dream work.*
+
+*Topics for breakout / Names*
+
+*
+*
+
+### If you choose to stay in the main room
+
+* There will always be a facilitator on hand in the main room to welcome new people and talk to anyone. 
+* You can come back here any time if you like and stay as long as you like.
+* Please ask any questions, explore the project more, or make any requests for adjustments which would make it easier to take part. 
+* You can choose which group to join and you can change your mind at any time.
+* A moderator can help you choose a group to join or an activity to take part in based on your skills and interests if you are unsure.
+* Join back here for group discussion
+
+------
+
+## Explore Current Activities in AutSPACEs
+
+If you're wondering where to start, check out some of the current activities that people are leading. You are welcome to join them, or propose a new task. 
+
+### Comms, engagement, and promotion
+
+*Suggested activities*  
+
+**Sign up below** 
+*Please add your name below if you are or would like to be involved in these activities (completely optional!)*  
```

Comment:
I think you should add at least one * (bullet point) below to start the list of names for each of the groups and then others can copy this tp add more. People might not be familiar with using hackmd.

Reply:
Good idea Emma 💛